benchmarks/models.py

```
@@ -0,0 +1,362 @@
+    raise ValueError(
+        "model manifest must define path, model_path, artifact_path, artifact, or model_id"
+    )
+
+
+def _validate_generation(generation: dict[str, Any]) -> tuple[int, float, float]:
+    """Validate and normalize generation options."""
+    max_tokens = generation.get("max_tokens", 32)
+    try:
+        max_tokens = int(max_tokens)
+    except (TypeError, ValueError) as e:
+        raise ValueError(f"max_tokens must be an integer, got {max_tokens!r}") from e
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)